colossus-metrics/src/main/scala/colossus/metrics/collectors/Gauge.scala

```diff
@@ -76,8 +76,9 @@ class ConcreteGauge(params: GaugeParams) extends Gauge with LocalLocality with T
     }
   }
 
+  //TODO: how gauges are aggregated should be configurable, since it's not obvious which should be the default
```

Comment:
Yea..I can't think of a better way of doing it..and I think making it configurable is the only real option.